sphinx_airflow_theme/sphinx_airflow_theme/theme.conf

```diff
@@ -6,3 +6,6 @@ pygments_style = default
 [options]
 canonical_url =
 analytics_id =
+# Default set by python code, need to list this here to avoid warning from Sphinx
+navbar_links =
```

Comment:
How can I set a dictionary here? Can you update docs? https://github.com/apache/airflow-site/tree/master/sphinx_airflow_theme#configuration

Reply:
Will update docs, you can't specify dict in here (but shpinx warns if this isn't here), so you have to set it via conf.py in end user (airflow)

Reply:
Done, PTAL.